src/test/java/org/hisp/dhis/EventTest.java

```diff
@@ -136,10 +136,10 @@ void testSerializeOccurredAt() {
         "programStage":"Zj7UnCAulEk",\
         "orgUnit":"DiszpKrYNg8",\
         "status":"COMPLETED",\
-        "createdAt":"2025-03-10T13:35:22.314",\
-        "updatedAt":"2025-03-10T13:35:22.314",\
-        "occurredAt":"2025-03-10T13:35:22.314",\
-        "completedAt":"2025-03-10T13:35:22.314",\
+        "createdAt":"2025-03-10T14:35:22.314",\
```

Comment:
Could you double check this one? I suspect we need to set UTC time zone explicitly. You are one hour behind me so this looks like a time zone issue.